### PR TITLE
HS Registered Animation TM-2264

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
- [x] Def is not going to work at different resolutions, so must address that.
- [x] A bit off-center
- [x] turn off for condensed bid tracker
- [x] replace hs status icons with hs register ribbon
- [x] Bureau Position Manager - add hs animation to hs register ribbon
- [x] Bureau Position Manager - add hs animation that does not have hs register ribbon

| Feature        | Browser           | Good  |
| ------------- |:-------------:|:-----:|
|Bid Tracker| Chrome |yes|
|| IE11 ||
|| safari |yes|
|| firefox |yes|
||edge|yes|
|PMD Bidder| Chrome ||
|| IE11 ||
|| safari ||
|| firefox |  |
||edge||

~Waiting on feedback to consolidate and clean-up the code.~ Done

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
hs
Proof it works, before I break it

File: src/Components/BidTracker/BidPhaseConfig/Complete.js [hunk 1]
```diff
@@ -242,7 +242,7 @@ export default function bidClassesFromCurrentStatus(bid = { status: 'draft' }) {
         number: APPROVED_NUMBER,
         tooltip: APPROVED_TOOLTIP };
       return bidClassObject;
-    // When the handshake is accepted, we display an overlay alert, so it can look the same
+    // When the handshake is registered, we display an overlay alert, so it can look the same
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/BidPhaseConfig/Complete.js:245]: Gonna start renaming these when I see them, now that TM has native HS functionality too

File: src/Components/BidTracker/BidStep/BidSteps.jsx [hunk 2]
```diff
@@ -53,6 +56,16 @@ const BidSteps = (props, context) => {
         </ConfettiIcon>
       );
     }
+    if (includes(bidPropsAfterRegister, bid.status) &&
+      bidData[status.prop].title[0] === GET_HAND_SHAKE_COMPLETE_REGISTER_TITLE()[0]) {
+    //  if we ONLY want to show the animation right after the step
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/BidStep/BidSteps.jsx:61]: Right now, animation is showing for all statuses after HS Registered, but if we just want to show it for just after they received, then this handles that

File: src/Components/BidTracker/BidStep/HandshakeAnimation.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            <FA className={`left-initial ${animate ? 'animate-left-initial' : ''}`} name="hand-paper-o" />
+            <FontAwesomeIcon className={`left-secondary ${animate ? 'animate-left-secondary' : ''}`} icon={faHandPaper} />
+          </div>
+          <div className={`right ${animate ? 'animate-right' : ''}`}>
+            <FA className={`right-initial ${animate ? 'animate-right-initial' : ''}`} name="hand-paper-o" />
+            <FontAwesomeIcon className={`right-secondary ${animate ? 'animate-right-secondary' : ''}`} icon={faHandPaper} />
+          </div>
+          <FA className={`hs-bidder ${animate ? 'animate-hs-bidder' : ''}`} name="handshake-o" />
+          <FA className="oc-icon" name="question-circle" />
+        </div>
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/BidStep/HandshakeAnimation.jsx]: unfortunately there is a lot of code repetition here. I tried to separate them out/reduce the repetition, but bc the bidder animation includes its starting icon, it complicates it a bit for the ribbon

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**